src/main/java/app/config/ApplicationConfiguration.java

```diff
@@ -50,6 +51,9 @@ public RequestCallerService getCaller(){
     @Bean
     public CryptoService getCryptoService(){ return new CryptoService();}
 
+    @Bean
```

Comment:
No instance of CPXKey needed. The methods in there are static so that you can use them without creating an instance